libs/wingsdk/src/cloud/api.ts

```diff
@@ -183,10 +183,50 @@ export class Api extends Resource {
    * @returns OpenAPI path
    * @internal
    */
-  public static _toOpenApiPath(path: string) {
+  public static renderOpenApiPath(path: string) {
     return path.replace(/\/:([A-Za-z0-9_-]+)/g, "/{$1}");
   }
 
+  /**
+   * Generates an object containing default CORS response headers and OPTIONS response headers.
+   * @param corsOptions The CORS options to generate the headers from.
+   * @returns An object containing default CORS response headers and OPTIONS response headers.
+   */
+  public static renderCorsHeaders(
+    corsOptions?: ApiCorsOptions
+  ): CorsHeaders | undefined {
```

Comment:
This `CorsHeader` is a pain to use with JSII since only string-indexed map types are supported.

Reply:
We can refactor this type later